src/components/apps/savedLaunch/SavedLaunchListing.js

```diff
@@ -381,6 +433,49 @@ function ListSavedLaunches(props) {
                         </IconButton>
                     </DialogTitle>
                     <DialogContent>
+                        {!!selectedIL ? (
+                            <FormControlLabel
+                                disabled={!selectedIL}
+                                control={
+                                    <Switch
+                                        size="small"
+                                        checked={useInstantLaunch}
+                                        onChange={(event) =>
+                                            setUseInstantLaunch(
+                                                event.target.checked
+                                            )
+                                        }
+                                        name={t(
+                                            "savedLaunchEmbedUseInstantLaunch"
+                                        )}
+                                    />
+                                }
+                                label={
+                                    <Typography variant="body2">
+                                        {t("savedLaunchEmbedUseInstantLaunch")}
+                                    </Typography>
+                                }
+                            />
+                        ) : (
+                            <Button
+                                variant="contained"
+                                startIcon={<Add />}
+                                onClick={(event) => {
+                                    event.stopPropagation();
+                                    event.preventDefault();
```

Comment:
I don't think these `event` calls should be required here. Usually this is only needed if the Button is on top/inside of something else that is intercepting clicks, like maybe an `AccordionSummary` or `MenuItem`  🤔

Reply:
Hm, that's fair, though I kind of feel like keeping it here because then it's more insurance in case this at some point _does_ end up within something that might care, but if there's a reason to remove it I'm happy to do that as well.